src/pages/pricing/00.pricing-page.tsx

```
@@ -299,47 +524,351 @@ export function PricingPage({ profile: _profile }: PricingPageProps) {
+                    }`}
+                  >
+                    ₱{amount}
+                  </button>
+                ))}
+              </div>
+              
+              <div className="mb-4">
+                <label className="block text-sm font-light text-white/70 mb-2">
+                  Or enter custom amount
+                </label>
+                <input
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)